estela-api/api/mixins.py

```diff
@@ -19,3 +20,9 @@ class BaseViewSet(viewsets.GenericViewSet):
     authentication_classes = [TokenAuthentication]
     permission_classes = [IsAuthenticated, IsProjectUser, IsAdminOrReadOnly]
     pagination_class = APIPageNumberPagination
+
+
+class ActivityHandler(viewsets.GenericViewSet):
```

Comment:
Please change this to `class ActivityHandlerMixin:`. Inheriting from `viewsets.GenericViewSet` should not be needed here. Remember to change all the occurrences of the name. Let me know if there are any problems with this change.